gcp/databases/cloudsql/private_postgres/versions.tf

```diff
@@ -0,0 +1,29 @@
+terraform {
+  required_version = ">= 0.13"
+  required_providers {
+    null = {
+      source  = "hashicorp/null"
+      version = "~> 3.1.0"
+    }
+    random = {
+      source  = "hashicorp/random"
+      version = "~> 3.1"
+    }
+    google = {
+      source  = "hashicorp/google"
+      version = ">= 4.4.0, < 5.0"
+    }
+    google-beta = {
```

Comment:
What needs beta? Add some comment in the code as to why.

Reply:
Not needed